retirement/templates/reserved_place.html

```diff
@@ -1,90 +1,128 @@
-        margin: 20px;
-        text-align: center;
-      }
+        .button:hover {
+            color: #142823;
+            background-color: transparent;
+            cursor: pointer;
+        }
 
-      .email__logo img {
-        height: 60px;
-        margin: auto;
-      }
+        .email {
+            background-color: #F7EDD9;
+            height: auto;
+            padding: 20px;
+        }
 
-      .email__content {
-        background-color: #FCF7EF;
-        max-width: 600px;
-        padding: 20px;
-        margin: auto;
-      }
+        .email__logo {
+            margin: 20px;
+            text-align: center;
+        }
 
+        .email__logo img {
+            height: 60px;
+            margin: auto;
+        }
 
-      .email__content__title {
-        font-size: 24px;
-        font-weight: 900;
-      }
+        .email__content {
+            background-color: #FCF7EF;
+            max-width: 600px;
+            padding: 20px;
+            margin: auto;
+        }
 
-      .email__content__subtitle {
-        font-size: 18px;
-        margin-top: 50px;
-      }
+
+        .email__content__title {
+            font-size: 24px;
+            font-weight: 900;
+        }
+
+        .email__content__subtitle {
+            font-size: 18px;
+            margin-top: 50px;
+        }
+
+        .email__content__separator {
+            border-top: 1px solid #0f2822;
+            margin-top: 60px;
+        }
+
+        .email__content__button {
+            padding-top: 60px;
+        }
 
     </style>
-  </head>
-  <body>
-    <div class="email">
-      <div class="email__logo">
+</head>
+<body>
+<div class="email">
+    <div class="email__logo">
         {% load static %}
-        <img src="{% static 'images/logo_general.png'%}" alt="">
-      </div>
-      <div class="email__content">
+        <img src="{% static 'images/logo_general.png' %}" alt="">
+    </div>
+    <div class="email__content">
         <h1 class="email__content__title">
-          Exclusivité 24h
+            Exclusivité 24h
         </h1>
         <p>
-          Des places se sont libérées pour une retraite à laquelle
-          vous êtes intéressés. Une place a été réservée pour vous
-          pour les prochaines 24 heures. Faites vite!
+            Des places se sont libérées pour une retraite à laquelle
+            vous êtes intéressés. Une place a été réservée pour vous
+            pour les prochaines 24 heures. Faites vite!
         </p>
         <h3>
-          Retraite: {{RETREAT_NAME}}
+            Retraite: {{ RETREAT_NAME }}
         </h3>
         <p>
-          Vous pouvez accéder à cette place exclusive en vous connectant
-          à la plateforme Thèsez-vous!
+            Vous pouvez accéder à cette place exclusive en vous connectant
+            à la plateforme Thèsez-vous!
+        </p>
+        <p>
+            Vous pouvez vous désinscrire de la file d'attente avec ce <a href="{{ WAITE_QUEUE_URL }}">
+                lien
```

Comment:
After some looks on other platform i think we can just create a link with this sentence `Se désinscrire de la file d'attente` 

Centered horizontally 

Note: we will need to think about the I18N for the content of the email... we completely forgot that part.

Reply:
![image](https://user-images.githubusercontent.com/11727354/62358470-1606dd00-b515-11e9-8435-8983a9e52c1d.png)